yarn.lock

```diff
@@ -13023,10 +13023,10 @@ sodium-native@^3.3.0:
   version "0.2.0"
   resolved "git+https://github.com/stellar/solar-stellarorg#78ce86e856d21e77fab77b3767c1a224bfb7e9fe"
 
-soroban-client@^0.4.0:
-  version "0.4.0"
-  resolved "https://registry.yarnpkg.com/soroban-client/-/soroban-client-0.4.0.tgz#738a402ab440cd9c34d14380150192f309866263"
-  integrity sha512-4t+SioFeUnG9dptkEzRzSgFKXTuRMz1abtxmzE461QVUCLIDayMIjY8B3vloKiWNSx9y/wNBIA5IA8PtAIm8rQ==
+soroban-client@^0.5.0:
```

Comment:
looks like I forget to commit an updated yarn.lock for the previous soroban-client upgrade